packages/block-editor/src/components/block-alignment-matrix-toolbar/README.md

```diff
@@ -0,0 +1,72 @@
+# Alignment Matrix Toolbar
+The alignment matrix toolbar allows users to quickly adjust inner block alignment; this is in contrast to the alignment toolbar that aligns the frame block.
```

Comment:
```suggestion

The alignment matrix toolbar allows users to quickly adjust inner block alignment; this is in contrast to the alignment toolbar that aligns the frame block.
```

Maybe just an empty line after the title.

Reply:
Sure